classes/war/WarAction.php

```diff
@@ -283,42 +295,19 @@ public function progressActiveOperation(): string
                     $caravan_resources = json_encode($caravan_resources);
                     $this->system->db->query("UPDATE `caravans` SET `resources` = '{$caravan_resources}' WHERE `id` = {$this->target_id}");
                     break;
-                case self::OPERATION_LOOT_TOWN:
```

Comment:
Is this getting removed or is it moved somewhere else?

Reply:
Looting towns is no longer a feature since you can take them directly.